templates/coderd.yaml

```diff
@@ -27,6 +27,9 @@ spec:
         app.kubernetes.io/part-of: coder
         app: {{ include "coder.serviceName" . }}
         coder.deployment: {{ include "coder.serviceName" . }}
+        {{- with .Values.coderd.extraLabels -}}
```

Comment:
I don't see any unit tests for this :stuck_out_tongue:

Reply:
Your unit tests caught the lack of unit tests! 🎉